src/Providers/DataGrid/Wijmo/Grid/FlexGrid.ts

```diff
@@ -84,6 +85,14 @@ namespace Providers.DataGrid.Wijmo.Grid {
             return this.config.getProviderConfig();
         }
 
+        private _updateColumnWidth(
+            grid: wijmo.grid.FlexGrid,
+            event: wijmo.grid.CellRangeEventArgs
+        ): void {
+            const column = event.getColumn();
+            this.getColumn(column.binding).config.width = column.width;
```

Comment:
Shouldn't we protect this code a bit more? I'm open to a no, it's just a question!